src/Composition/scene.jl

```diff
@@ -308,7 +308,7 @@ Defines global properties of the system, such as the gravity field. Exactly one
 | `gravityField`                  | [`UniformGravityField`](@ref)`()`       |
 | `useOptimizedStructure`         | true                                    |
 | `enableContactDetection`        | true                                    |
-| `contactDetection`              | [`ContactDetectionMPR_handler`](@ref)`()`|
+| `mpr_tol_rel`                   | 1.0e-20                                 |
```

Comment:
`mprTolerance` or so would be more consistent. "Relative" could be explained in docu.

Reply:
I like your suggestion. I'll change that.